### ISSUE TITLE
ADUNs started in 2013 missing Org links

### ISSUE THREAD

**Issue by leowmjw** [Sinar/popit_ng#143]
ADUNs that started in 2013 missing their Organization link as per below; added the Post_ID for easier cross reference

https://gist.github.com/leowmjw/32bba9674f46a4699d0e288a97aff797

**Comment by sweemeng** [comment]
http://api.popit.sinarproject.org/en/posts/545f7f865222837c2c05b8e6

post have org linked on API. Will look at search side

**Comment by leowmjw** [comment]
@sweemeng The data is in posts but not in memberships.  So can you confirm that this is by design; and there should not be organization, organization_id field in memberships.

This is needed because will need to check memberships that start/end at a certain date range.  For example; show all ADUNs since 2013 (inclusive).  If no organization info; then will need to do a further JOIN / lookup from posts.

See below for example when extracted via memberships resource end point:
```
_key:545f82355222837c2c05b9a3

	data		{21}
	
organization	:	null
area	:	null
organization_id	:	null
	
member	:	null
	
member_id	:	null
	
on_behalf_of	:	null
	
on_behalf_of_id	:	null
	
area_id	:	null
	
label	:	null
	
language_code	:	en
	
start_date	:	2004
	
end_date	:	2008
	
created_at	:	2015-12-21T02:55:09.374273Z
	
updated_at	:	2015-12-21T02:55:09.374315Z
	
person_id	:	545e46e25222837c2c0592ce
	
post_id	:	545f7f855222837c2c05b8dc
	
role	:	Member of State Assembly

...
```

**Comment by sweemeng** [comment]
It is originally designed this way, but i am adding an improvement to ensure this field is populated if post have org linked. Very useful for search.

**Comment by sweemeng** [comment]
implemented in #147 
Anytime a membership only created with post and person, and post is linked to organization. Membership organization field will use post organization. If membership is update, and organization is null/None, and post have organization linked, membership will link to that organization. 

BTW, if you supply organization on create and update, they will not override the field. 

I added a script to convert the existing data. 

Please verify

**Comment by leowmjw** [comment]
@sweemeng Confirmed fixed.  I have included below the AQL used for later when I give you access to my c9.io workspace which will have the scripts needed to reproduce/test out any of the data combinations from the PopIt API

```
FOR p IN popit_posts
    // FILTER LOWER(p.data.area.state) == 'selangor' && LIKE(LOWER(p.data.role),'%member of parliament%')
    FILTER LIKE(LOWER(p.data.role),'%state%')
    FOR m IN memberships
        FILTER m.data.post.id == p._key
        FILTER DATE_TIMESTAMP(m.data.start_date) >= DATE_TIMESTAMP("2013")
        FILTER m.data.organization.name == null
        SORT p.data.area.id, DATE_TIMESTAMP(m.data.start_date)
        RETURN { 
            myname: m.data.person.name, 
            myorg: [ m.data.organization.id, m.data.organization.name ], 
            mypost: [ m.data.post.id, m.data.post.label ],
            mystart: m.data.start_date,
            myarea: p.data.area
            }

```
